### PR TITLE
remove inline styles

### PR DESCRIPTION
**Pull request by MeenaModhvadia**
## Description
The styling of all our applications should be read from a single cascading style sheet.  This allows for a consistent look for all the applications elements in one place.  There are at least 8 instances of ‘inline’ styles, styles that have been hard coded into different pages.  These need to be moved into the a stylesheet if they are not already there.

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [ ] (3) Tests passing
* [ ] (4) Branch ready to be merged (not work in progress)
* [ ] (5) No superfluous changes in diff
* [ ] (6) No TODO's without new ticket numbers
* [ ] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`
* [ ] (8) Data migration script is created if any of letter templates is changed

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CDPT-697?atlOrigin=eyJpIjoiNzRmY2E1YjM0ODk1NDcwMmE4YzIxNDU2OTBjZmViZDEiLCJwIjoiaiJ9

### CODE REVIEW

File: app/views/cases/offender_sar_complaint/_exempt_pages_step.html.slim [hunk 1]
```diff
@@ -15,7 +15,7 @@ div class="case-#{@correspondence_type_key}"
   / this does not appear in the list of steps for new cast creation as found in
   / OffenderSARCaseForm -> def steps method
   = form_for @case, url: case_sar_offender_complaint_path(@case), as: :offender_sar_complaint do |f|
-    = f.number_field :number_exempt_pages, style: 'width: 10%'
+    = f.number_field :number_exempt_pages, {class: 'button pages_number'}
```
Comment (vertism) [inline app/views/cases/offender_sar_complaint/_exempt_pages_step.html.slim:18]: Should this have the button class?

Comment (MeenaModhvadia) [reply app/views/cases/offender_sar_complaint/_exempt_pages_step.html.slim:18]: Thank you - removed this.

Comment (vertism) [reply app/views/cases/offender_sar_complaint/_exempt_pages_step.html.slim:18]: Have you pushed your change?

**Review by vertism — APPROVED** [review]
Looks good!

**Pull request merged**